plugin/gatsby-browser.js

```diff
@@ -3,9 +3,9 @@
 import confetti from "canvas-confetti";
 
 export const onInitialClientRender = (_, options) => {
-  const { colors = ["#ffffff"] } = options;
+  const { colors = ["#ffffff"], intensity, duration } = options;
```

Comment:
You also need to give duration a default of 15 seconds here.

Why, as is should come through the plugin schema set up? The schema code in gatsby-node is skipped in Gatsby v2 and we are supporting v2.

Reply:
You so so by adding "= 15 * 1000" after duration.

Reply:
Never mind, we'll make Gatsby v2.4.0 required.

Reply:
Waaait? I a NOT doing `= 15 * 1000`
😸

Reply:
No need, we are requiring version 2.4 that has plugin schema, so we are sure to get a default value 🎉